src/components/ArticleContent/ArticleContent.js

```diff
@@ -326,7 +332,8 @@ ArticleContent.propTypes = {
   */
 const mapStateToProps = state => ({
   article: state.oneArticleReducer.article,
-  loading: state.oneArticleReducer.loading
+  loading: state.oneArticleReducer.loading,
+  reload: state.notification.reload
```

Comment:
Can you please explain what this state `reload` does?

Reply:
I am also seeing that reload is not defined anywhere in your initial state. 
Could you please explain what the value of the initial state of `reload` is and what's is expected to be stored in it?